### PR TITLE
Estimation for next month

### PR DESCRIPTION
**Pull request by FredrikAreschoug**

### CODE REVIEW

File: src/main/kotlin/com/hedvig/accountservice/graphql/resolvers/Query.kt [hunk 1]
```diff
@@ -49,7 +50,6 @@ class Query(
 
     fun chargeEstimation(env: DataFetchingEnvironment): ChargeEstimation {
         val memberId = env.getToken()
-        // TODO: Return null when >= 27th and before the 1st?
```
Comment (palmenhq) [inline src/main/kotlin/com/hedvig/accountservice/graphql/resolvers/Query.kt]: nice catch

File: src/main/kotlin/com/hedvig/accountservice/graphql/resolvers/Query.kt [hunk 2]
```diff
@@ -74,18 +74,19 @@ class Query(
...
     private fun getEstimatedDiscount(memberId: String): MonetaryAmount {
-        val now = YearMonth.now()
-        val currentDiscounts = productPricingService.getMonthlyCampaignDiscounts(memberId, now)
+        val currentDiscounts = productPricingService.getMonthlyCampaignDiscounts(memberId, getMonthForEstimation())
         return currentDiscounts
             .map { discount -> discount.amount }
             .fold(Money.of(0, "SEK")) { total, discount -> total.add(discount) }
     }
+
+    private fun getMonthForEstimation() =
```
Comment (palmenhq) [inline src/main/kotlin/com/hedvig/accountservice/graphql/resolvers/Query.kt]: idk if i like putting this logic in the query resolver but i guess its ok

File: src/main/kotlin/com/hedvig/accountservice/graphql/resolvers/Query.kt (hunk 1 above)
Comment (vonElfvin) [reply src/main/kotlin/com/hedvig/accountservice/graphql/resolvers/Query.kt]: This comment was placed to remind us of the issue this PR is here to fix 😇

File: src/main/kotlin/com/hedvig/accountservice/graphql/resolvers/Query.kt [hunk 3]
```diff
@@ -74,18 +74,19 @@ class Query(
     }
 
     private fun getEstimatedSubscription(memberId: String): MonetaryAmount {
-        val now = YearMonth.now()
-        val currentProductSubscriptions = productPricingService.getProductSubscriptions(memberId, now)
+        val currentProductSubscriptions = productPricingService.getProductSubscriptions(memberId, getMonthForEstimation())
```
Comment (vonElfvin) [inline src/main/kotlin/com/hedvig/accountservice/graphql/resolvers/Query.kt]: I'm personally a fan of putting variables outside function calls i.e. 
```suggestion
        val monthForEstimation = getMonthForEstimation()
        val currentProductSubscriptions = productPricingService.getProductSubscriptions(memberId, monthForEstimation)
```
to make the code clear. This is just an opinion though so it's nothing worth fixing.

File: src/main/kotlin/com/hedvig/accountservice/graphql/resolvers/Query.kt (hunk 1 above)
Comment (palmenhq) [reply src/main/kotlin/com/hedvig/accountservice/graphql/resolvers/Query.kt]: Yes that I understand, but i would've forgotten to remove it haha 😄

**Pull request merged**